addons/web/static/src/core/autocomplete/autocomplete.js

```diff
@@ -228,6 +228,12 @@ export class AutoComplete extends Component {
         this.open(true);
     }
 
+    onInputFocus() {
+        if (this.inputRef.el) {
```

Comment:
For my knowledge: In which case is this condition needed ?

Reply:
True. I can remove the condition because the input ref el is always present.